frontend/src/views/WheresMyGene/components/Filters/components/Compare/connect.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  DefaultDropdownMenuOption,
+  InputDropdownProps as IInputDropdownProps,
+} from "@czi-sds/components";
+import { useCallback, useContext, useMemo } from "react";
+import { track } from "src/common/analytics";
+import { EVENTS } from "src/common/analytics/events";
+import { COMPARE_OPTIONS } from "src/views/WheresMyGene/common/constants";
+import {
+  DispatchContext,
+  State,
+  StateContext,
+} from "src/views/WheresMyGene/common/store";
+import { selectCompare } from "src/views/WheresMyGene/common/store/actions";
+import { Props } from "./types";
+
+export const useConnect = ({ areFiltersDisabled }: Props) => {
+  const dispatch = useContext(DispatchContext);
+  const { compare } = useContext(StateContext);
+
+  const DEFAULT_INPUT_DROPDOWN_PROPS: Partial<IInputDropdownProps> = useMemo(
```

Comment:
since this doesn't depend on any function level variables, we can just hoist this out as a top level constant?

```ts
const DEFAULT_INPUT_DROPDOWN_PROPS: Partial<IInputDropdownProps> = {
  sdsStyle: "square",
};
```

Reply:
<img width="702" alt="Screenshot 2023-09-28 at 1 04 24 PM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/7976579/4181b716-3e25-4650-aa24-2a0053067d11">
this is why